src/main/resources/lib/credentials/select/select.js

```diff
@@ -200,6 +200,17 @@ Behaviour.specify("BUTTON.credentials-add-menu", 'credentials-select', -99, func
             }
             window.credentials.add(item.srcElement.getAttribute('data-url'));
         });
+        // YUI menu will not parse disabled when using DIV-LI only when using SELECT-OPTION
```

Comment:
Oof. Will we ever be able to replace YUI without having to go all the way to Blue Ocean?